examples/docs_snippets/docs_snippets_tests/conftest.py

```diff
@@ -26,6 +29,13 @@ def mock_s3_bucket(mock_s3_resource):
 # ########################
 
 
+# Runs once before every test
+def pytest_configure():
+    # Disable the dg update check for all tests because we don't want to bomb the PyPI API.
+    # Tests that specifically want to test the update check should set this env var to "1".
+    os.environ[DG_UPDATE_CHECK_ENABLED_ENV_VAR] = "0"
```

Comment:
We also probably want to disable this behavior when running in automation (e.g. CI) - wonder if there is a way we can do so proactively rather than potentially spamming pypi (or e.g. being unable to hit pypi if the org has gated network access and a pypi pullthrough cache or something)